### PR TITLE
Document object_ma1 (Romani)

### PR DESCRIPTION
**Pull request by tom-overton**
I was under the mistaken impression that object_uch was the last object in Invadepoh that needed to be documented, but I was apparently wrong. object_ma1 was one of those handful of objects that had a limb enum and nothing else, so I went and documented the object fully.

### CODE REVIEW

**Comment by tom-overton** [comment]
Whoops, Anghelo had this reserved and I didn't notice! Apologies, will close this one.

**Comment by tom-overton** [comment]
Reopening because @AngheloAlf said it's fine if I do this. Please wait for their approval before merging this.

File: assets/xml/objects/object_ma1.xml
Comment (tom-overton) [reply assets/xml/objects/object_ma1.xml:31]: Most of the animations *aren't* written as gerund; this one just happens to be right next to animations like `gRomaniSittingAnim`. This is because I thought `gRomaniSitAnim` kind of looked weird. Would you be okay with me changing the following three animations to:
`gRomaniSitAnim`
`gRomaniSitHeadShakeAnim`
`gRomaniSitLookDownAnim`

Or would you prefer me make everything written as gerund?

Comment (AngheloAlf) [reply assets/xml/objects/object_ma1.xml:31]: Ohh, I see.
In that case I think this way is fine, no need to worry

File: src/overlays/actors/ovl_En_Ma4/z_en_ma4.c
Comment (hensldm) [inline src/overlays/actors/ovl_En_Ma4/z_en_ma4.c:301]:
```suggestion
    if (this->skelAnime.animation == &gRomaniRunAnim) {
```

**Review by hensldm — APPROVED** [review]

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**
